modules/virt-editing-vmis-web.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/cnv-manage-vmis.adoc
+
+[id="virt-editing-vmis-web_{context}"]
+= Editing a standalone virtual machine instance using the web console
+
+You can edit labels and annotations for a standalone virtual machine instance using the web console. Other items displayed in the *Overview* page for a standalone virtual machine instance are not editable.
+
+.Procedure
+
+. Click *Workloads -> Virtual Machines*  from the side menu.
+
+. Select a virtual machine instance. The *Virtual Machine Instance* page displays.
```

Comment:
Miss `Overview`, it should be `Virtual Machine Instance Overview` page.

Reply:
Made the correction.